recipes/soplex/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import collect_libs, copy, get, replace_in_file
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+from os.path import join
+
+required_conan_version = ">=1.53.0"
+
+
+class SoPlexConan(ConanFile):
+    name = "soplex"
+    description = "SoPlex linear programming solver"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://soplex.zib.de"
+    topics = ("simplex", "solver", "linear", "programming")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "BOOST": [True, False],
+        "GMP": [True, False],
```

Comment:
Options should be lower case and we generally prefix them `with_` when they added dependencies 

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#options

Reply:
I changed the format now.